packages/env/lib/spawn-docker-compose-run-command.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+const { spawn } = require( 'child_process' );
+
+/**
+ * @typedef {import('../config').Config} Config
+ */
+
+/**
+ * Runs an arbitrary command on the given Docker container.
+ *
+ * @param {Config}      config     The wp-env configuration.
+ * @param {string}      container  The Docker container to run the command on.
+ * @param {string|null} workingDir The working directory that the command should be executed in.
+ * @param {string[]}    command    The command to run.
+ * @param {Object}      spinner    A CLI spinner which indicates progress.
+ */
+module.exports = function spawnDockerComposeRunCommand(
+	config,
+	container,
+	workingDir,
+	command,
+	spinner
+) {
+	// Start building the commany we're going to send to Docker.
+	const composeCommand = [ '-f', config.dockerComposeConfigPath, 'run' ];
+
+	// Pass the working directory if one has been supplied.
+	if ( workingDir ) {
+		composeCommand.push( '-w', workingDir );
+	}
+
+	// Finish building the command.
+	composeCommand.push( '--rm', container, ...command );
```

Comment:
In the previous implementation, we had to split the command on spaces. Is that still required?

Reply:
Since we're calling `spawn` with the `"shell"` option set to true, all of this is being passed to a shell to execute instead of using a system call to start the executable. Technically that means it doesn't really matter, but, arguably, we don't need to use `"shell": true` in the first place. I've gone ahead and changed this behavior though for the sake of consistency.